modules/comment/comment.resolver.js

```diff
@@ -25,6 +25,19 @@ const commentsQuery = {
       ];
     }
   },
+
+  getAllCommentsByUser: async (parent, args) => {
+    try {
```

Comment:
Why do we need a `try {...} catch {...}` block here if neither in service nor in the controller, we throw any error?